### PR TITLE
Add Mangapure

### PR DESCRIPTION
**Pull request by MikeZeDev**
All Manganelos.com mangas redirect here, better adding mangapure and getting rid of manganelos

### CODE REVIEW

File: web/src/engine/websites/MangaPure.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    slug: string
+}
+
+@Madara.MangaCSS(/^{origin}\/read\/[^/]+$/, 'div.post-title')
+@Madara.MangasMultiPageCSS(undefined, undefined, '/popular-manga?page={page}')
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('mangapure', 'MangaPure (manganelos)', 'https://mangapure.net', Tags.Media.Manhwa, Tags.Media.Manhua, Tags.Media.Manga, Tags.Language.English, Tags.Source.Aggregator);
```
Comment (ronny1982) [inline web/src/engine/websites/MangaPure.ts:20]: Should there be a Bookmark migration from `manganelos` to `mangapure`?

File: web/src/engine/websites/MangaPure.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import icon from './MangaPure.webp';
+import type { Manga } from '../providers/MangaPlugin';
+import { Chapter, DecoratableMangaScraper, Page } from '../providers/MangaPlugin';
+import * as Madara from './decorators/WordPressMadara';
+import * as Common from './decorators/Common';
+import { FetchCSS, FetchRequest } from '../FetchProvider';
+
+type MangaID = {
+    post: string,
+    slug: string
```
Comment (ronny1982) [inline web/src/engine/websites/MangaPure.ts:11]: Unused member `slug`, may be removed from type definition.

File: web/src/engine/websites/MangaPure.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    public constructor() {
+        super('mangapure', 'MangaPure (manganelos)', 'https://mangapure.net', Tags.Media.Manhwa, Tags.Media.Manhua, Tags.Media.Manga, Tags.Language.English, Tags.Source.Aggregator);
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    public override async FetchChapters(manga: Manga): Promise<Chapter[]> {
+        const mangaid: MangaID = JSON.parse(manga.Identifier);
```
Comment (ronny1982) [inline web/src/engine/websites/MangaPure.ts:28]: May use decomposition to get member directly
```typescript
const { post } = Json.parse(manga.Identifier) as MangaID;
```

**Review by ronny1982 — APPROVED** [review]

File: web/src/engine/websites/MangaPure.ts (hunk 1 above)
Comment (MikeZeDev) [reply web/src/engine/websites/MangaPure.ts:20]: I asked myself but since its from AnyACG to Madara, maybe it just wont fit?

**Pull request merged**